command.go

```diff
@@ -1050,7 +1058,7 @@ func (c *Command) ExecuteC() (cmd *Command, err error) {
 			c = cmd
 		}
 		if !c.SilenceErrors {
-			c.PrintErrln("Error:", err.Error())
+			c.PrintErrln(c.errPrefix, err.Error())
```

Comment:
Won’t the new variable be empty by default?  We probably need a setter for it and make it default to “Error:”